examples/simple_zonal_with_asm/main.tf

```diff
@@ -14,10 +14,6 @@
  * limitations under the License.
  */
 
-locals {
-  cluster_type = "simple-zonal-asm"
```

Comment:
We can connect this to module.gke.name. The current name seems like a placeholder we forgot to remove.

Reply:
This isn't being used nor is there a variable by that name in the asm module. The name is already set as `module.gke.name` for the asm module.